### PR TITLE
bump containerd to v1.5.5 and kernel to mainline 5.14

### PR DESCRIPTION
**Pull request by chrischdi**

### CODE REVIEW

File: images/capi/ansible/roles/dhc/tasks/debian.yml [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
-    packages:
-      - linux-oem-20.04b
+- name: Install mainline Kernel
+  block:
+  - name: add mainline lts ppa
+    command: |
+      add-apt-repository -y ppa:tuxinvader/lts-mainline
+    become: true
+
+  - name: install kernel kernel from ppa
```
Comment (bavarianbidi) [inline images/capi/ansible/roles/dhc/tasks/debian.yml]:
```suggestion
  - name: install kernel from ppa
```

File: images/capi/packer/config/containerd.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-    "containerd_version": "1.5.4",
-    "containerd_sha256": "591e4e087ea2f5007e6c64deb382df58d419b7b6922eab45a1923d843d57615f",
+    "containerd_version": "1.5.5",
+    "containerd_sha256": "45f02cfc65db47cf088c95555906e1dcba7baf5a3fbad3d947dd6b9af476a144",
```
Comment (bavarianbidi) [inline images/capi/packer/config/containerd.json:3]: where do you get this sha256 from?

as we download the [`cri-containerd-cni`](https://github.com/c445/image-builder/blob/c445/images/capi/packer/qemu/packer.json#L11) binary we must use `45f02cfc65db47cf088c95555906e1dcba7baf5a3fbad3d947dd6b9af476a144` as sha256

**Review by bavarianbidi — CHANGES_REQUESTED** [review]

File: images/capi/packer/config/containerd.json (hunk 2 above)
Comment (chrischdi) [reply images/capi/packer/config/containerd.json:3]: this is exactly the sha which is set?! It is from https://github.com/containerd/containerd/releases/download/v1.5.5/cri-containerd-cni-1.5.5-linux-amd64.tar.gz.sha256sum

Comment (bavarianbidi) [reply images/capi/packer/config/containerd.json:3]: :see_no_evil: sorry for confusing you ... not sure where my minds already were when writing this comment

**Review by bavarianbidi — APPROVED** [review]

**Review by seanschneeweiss — APPROVED** [review]

**Pull request merged**